### ISSUE TITLE
FixMatch for Multi-class Classification

### ISSUE THREAD

**Issue by GuoJingtao-1997** [kekmodel/FixMatch-pytorch#47]
Thanks for sharing this excellent work. I just wonder if there is any idea to apply this algorithm for multi-class classification. Could I simply replace softmax with sigmoid to implement it?
